src/utils/HandleValidate.js

```diff
@@ -0,0 +1,15 @@
+const OutputView = require('../view/OutputView');
+
+const HandleValidate = {
+  checkValidate(validate, input) {
+    try {
+      validate(input);
+      return true;
+    } catch (error) {
+      OutputView.printErrorMessage(error);
+      return false;
+    }
+  },
+};
```

Comment:
함수만 작성해서 export 하는 것도 좋아보입니다.
```suggestion
const checkValidate = (validate, input) => {
  try {
    validate(input);
    return true;
  } catch (error) {
    OutputView.printErrorMessage(error);
    return false;
  }
}
```

Reply:
객체에 너무 집중한 나머지 무조건적으로 객체안에 넣는 습관이 있었던것 같습니다. 감사합니다 많이 배웁니다.